CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ if(APPLE)
 
   # iOS build options
   option(BUILD_IOS           "Build for iOS"               NO)
-  option(FORCE_RESET_OSX_DEPLOYMENT_TARGET "Clear the OSX Deployment Target Set" YES)
+  option(FORCE_RESET_OSX_DEPLOYMENT_TARGET "Clear the OSX Deployment Target Set" ${FORCE_RESET_DEPLOYMENT_TARGET})
```

Comment:
The cmake command can be invoked directly from shell/terminal, and in that case default value for FORCE_RESET_OSX_DEPLOYMENT_TARGET should remain unchanged. Probably change above code to:

```cmake
option(FORCE_RESET_OSX_DEPLOYMENT_TARGET "Clear the OSX Deployment Target Set" YES)
if (DEFINED FORCE_RESET_DEPLOYMENT_TARGET)
  set(FORCE_RESET_OSX_DEPLOYMENT_TARGET, $FORCE_RESET_DEPLOYMENT_TARGET)
endif()
```